docs/plugin-proposal-class-properties.md

```diff
@@ -85,6 +85,17 @@ require("@babel/core").transformSync("code", {
 
 When `true`, class properties are compiled to use an assignment expression instead of `Object.defineProperty`.
 
+> ⚠️ Consider migrate to top level [`setPublicClassFields` assumptions](/assumptions#setPublicClassFields)
```

Comment:
```suggestion
> ⚠️ Consider migrating to the top level [`setPublicClassFields`](/assumptions#setPublicClassFields) assumption
```

Do we need to put the version here?
Actually we should add that too https://babeljs.io/assumptions and https://babeljs.io/docs/en/options#assumptions? 7.13